pages/reference/base-images/base-images.md

```diff
@@ -108,92 +107,97 @@ CMD ["node", "main.js"]
+| -------- | ------- | -------------------------------------------- |
+| Node.js  | v13.5.0 | armv6, armv7hf, aarch64, amd64, i386         |
+| Python   | v2.7.17 | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
+| OpenJDK  | v11-jdk | armv7hf, aarch64, amd64, i386, armv6         |
+| Go       | 1.13.4  | armv7hf, aarch64, amd64, i386, armv6         |
+| Dotnet   | 3.1-sdk | armv7hf, aarch64, amd64                      |
 
-| Distribution | Default                | Supported Architectures                      |
-|---------|------------------------------|----------------------------------------------|
-| Debian  | Debian GNU/Linux 10 (Buster) | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
-| Alpine  | Alpine Linux v3.10            | armv6, armv7hf, aarch64, amd64, i386 		|
-| Ubuntu  | 18.04 LTS (Bionic Beaver)    | armv7hf, aarch64, amd64, i386                |
-| Fedora  | Fedora 30               | armv7hf, aarch64, amd64, i386                |
+#### Notes
 
-| Language | Default  	                  | Supported Architectures                      |
-|---------|------------------------------|----------------------------------------------|
-| Node.js | v13.1.0                      | armv6, armv7hf, aarch64, amd64, i386         |
-| Python  | v2.7.17                       | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
-| OpenJDK | v11-jdk                    | armv7hf, aarch64, amd64, i386, armv6         |
-| Go      | 1.13.4                       | armv7hf, aarch64, amd64, i386, armv6         |
-| Dotnet  | 3.1-sdk                       | armv7hf, aarch64, amd64                |
+Devices with a [device type](/reference/base-images/devicetypes/) of `raspberry-pi` (Raspberry Pi1 and Zero) will be built from `balenalib/rpi-raspbian` and will be [Raspbian]({{ $links.githubLibrary }}/resin-rpi-raspbian) base images. Raspberry Pi2 and Pi3 base images have the Raspbian package source added and Raspbian userland pre-installed.
 
-#### Notices
+Not all OS distro and language stack versions are compatible with each other. Notice that there are some combinations that are not available in the `balenalib` base images.
 
-Not all OS distro and language stack versions are compatible with each other, pleace notice that there are some combinations which are not available in the `balenalib` base images.
 - [Nodejs dropped 32-bit builds](https://github.com/nodejs/build/issues/885) a while ago so i386-based nodejs images (Debian, Fedora and Ubuntu) v8.x and v6.x are official. New series (v10.x and v12.x) are using unofficial builds so please notice when using these images.
 - armv6 binaries were officially dropped from v12 and v12 armv6 support is now considered unofficial.
 - The Nodejs v6.x and v8.x series are not available for i386 Alpine Linux base images v3.9 and edge as node crashes with segfault error, we are investigating the issue and will add them back as soon as the issue is resolved.
```

Comment:
Nitpick: Nodejs v6.x isn't included in the list of supported Node.js versions earlier on the page :wink:

Reply:
Thanks, I left this as while it isn't supported there are v6 base images available on Docker Hub.